### DISCUSSION TITLE
The content in https://dioxuslabs.com/docs/0.3/guide/en/ has not been linked to the latest documentation

### DISCUSSION THREAD

**Discussion by rikkaka** [DioxusLabs/dioxus#1136] [General]
I found there is something outdated in https://dioxuslabs.com/docs/0.3/guide/en/getting_started/ssr.html. When I tried to suggest an edit via the upper right button, it was linked to an non-existent section in the repository.
<img width="943" alt="image" src="https://github.com/DioxusLabs/dioxus/assets/39625849/8a7063a2-b997-4d8b-a4a1-eb7f00928382">

I found the real path should be [dioxus](https://github.com/DioxusLabs/dioxus/tree/master)/[docs](https://github.com/DioxusLabs/dioxus/tree/master/docs)/[guide](https://github.com/DioxusLabs/dioxus/tree/master/docs/guide)/[src](https://github.com/DioxusLabs/dioxus/tree/master/docs/guide/src)/[en](https://github.com/DioxusLabs/dioxus/tree/master/docs/guide/src/en)/[getting_started](https://github.com/DioxusLabs/dioxus/tree/master/docs/guide/src/en/getting_started)/ssr.md now. The structure of the repository was renewed while the link in the docs has not been renewed yet.

Is there anyone who can update it?

**Comment by ealmloff** [comment]
That url is set here: https://github.com/DioxusLabs/dioxus/blob/master/docs/guide/book.toml#L21, but a lot of the docs will change with https://github.com/DioxusLabs/docsite/pull/63
